apps/slackbot/bolt_app.py

```diff
@@ -145,22 +151,29 @@ def log_formatter(logger):
             formatted_reply = f"""-- Step: {log["Step"]} -- \nThoughts: \n {reply["thoughts"]} """
             
             if "command" in reply: # add command if it exists
-                formatted_reply += f"""\nCommand: \n {reply["command"]}"""
+              formatted_reply += f"""\nCommand: \n {reply["command"]}"""
             
             log_strings.append(formatted_reply)
 
         else: # for final response
             formatted_reply = f"""-- Step: {log["Step"]} -- \nFinal Response: \n {reply}"""
             log_strings.append(formatted_reply)
```

Comment:
It seems the `-verbosex` option is resulting `None` as the response because of a missing return value here

Reply:
Okay, good thing you caught that error, I just fixed it in the next commit